ftests/ftests-nocontainer.sh

```diff
@@ -1,2 +1,25 @@
 #!/bin/bash
-./ftests.py -l 10 -L ftests-nocontainer.log --no-container
+
+START_DIR=$PWD
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+if [ "$START_DIR" != "$SCRIPT_DIR" ]; then
+	cp $SCRIPT_DIR/*.py $START_DIR
+fi
+
+if [ -d ../../src/python/build/lib.* ]; then
+	pushd ../../src/python/build/lib.*
+	export PYTHONPATH=$PYTHONPATH:$(pwd)
+	popd
+fi
```

Comment:
@drakenclimber Thanks for working on the patch series.  It looks great, minor nitpick, can you double-quote the variables.